### PR TITLE
upgrade cypress version

### PR DESCRIPTION
**Pull request by KrupaPammi**

### CODE REVIEW

**Comment by lebaz20** [comment]
@KrupaPammi You're right, babel lib needed to be updated, pushed it now.

**Comment by lebaz20** [comment]
Sorry I shouldn't have missed that, working on it.

File: config/webpack.config.dev.js [hunk 1]
```diff
@@ -122,6 +122,7 @@ module.exports = {
             loader: require.resolve('babel-loader'),
             options: {
               cacheDirectory: true,
+              presets: ['react-app']
```
Comment (lebaz20) [inline config/webpack.config.dev.js:125]: This is what fixes the babel error, As you can see it is exactly the same as babel config in package.json. But, we have to put it here as well, as babel-loader doesn't recognize config in package.json unfortunately.

**Review by lebaz20 — APPROVED** [review]

File: config/webpack.config.dev.js (hunk 1 above)
Comment (KrupaPammi) [reply config/webpack.config.dev.js:125]: Nice one, thanks @lebaz20

**Pull request merged**